main_test.py

```diff
@@ -0,0 +1,42 @@
+from statistics import mean
+
+import numpy as np
+
+from gym_jadx.envs.jadx_env import JadxEnv
+import random
+import time
+
+env = JadxEnv()
```

Comment:
Again the same comment with `gym.make()`

Reply:
Changed in the last commit